mscr-ui/src/common/utils/has-permission.tsx

```diff
@@ -114,13 +125,54 @@ export function checkPermission({
     return true;
   }
 
+  console.log(rolesInTargetOrganizations);
+  // Return true if user has data model editor role in target organization
+  if (
+    rolesInTargetOrganizations?.includes('DATA_MODEL_EDITOR')||rolesInTargetOrganizations?.includes('ADMIN')
+  ) {
+    return true;
+  }
+  
+
   // Return true if user has admin role in target organization
   if (
-    rolesInOrganizations.includes('ADMIN') &&
+    rolesInOrganizations.includes('') &&
```

Comment:
What does this mean, when does rolesInOrganizations include empty string? Maybe this step is unnecessary because of the new role check above?